### ISSUE TITLE
Tiles incorrectly displayed on very long screens

### ISSUE THREAD

**Issue by laurentS** [mapswipe/mapswipe#240]
**Describe the bug**
A user reported a bug with the display of tiles in built area projects. See screenshot below. The tile at top right somehow moved to a 4th row at the bottom.

**App version**
`2.0.4(0)` on android

**To Reproduce**
1. Open any built area project
2. see the problem

**Screenshots**
![image](https://user-images.githubusercontent.com/1745184/85285199-fefc1100-b47f-11ea-8141-dd45735bab70.png)

**Smartphone (please complete the following information):**
 - Device: [OnePlus7T
 - OS: Android 10

**Additional context**
The screen size is 2400x1080.
Debug info give the following screen metrics in the app:
```
TILE_VIEW_HEIGHT: 895.89
SCREEN_WIDTH: 454.73
SCREEN_HEIGHT: 965.89
TILE_SIZE: 227.36
```
This seems to be caused by an incorrect flow configuration for the tiles.

**Comment by kmpoppe** [comment]
Merge with #241, so this Issue is done and could be closed, right?

**Comment by laurentS** [comment]
We try to close issues only once the corresponding features/bug has been merged into production (ie `master`)

**Comment by laurentS** [comment]
Fixed with release `2.0.5(0)`.
